salt/metalk8s/container-engine/containerd/installed.sls

```diff
@@ -95,3 +104,5 @@ Configure registry IP in containerd conf:
 
         [debug]
         level = "{{ 'debug' if metalk8s.debug else 'info' }}"
+    - watch_in:
+        - service: Ensure containerd running
```

Comment:
Shouldn't this cause a service restart?

Reply:
It does, it's why I use `watch_in` and not `require_in`

When you do a `service.running` and you watch a state that do some changes then the service is restarted

Reply:
Thanks!